central/cluster/datastore/datastore_impl.go

```diff
@@ -545,7 +548,11 @@ func (ds *datastoreImpl) postRemoveCluster(ctx context.Context, cluster *storage
 	ds.removeK8SRoles(ctx, cluster)
 	ds.removeRoleBindings(ctx, cluster)
 
-	// TODO: Apparently, cluster cves are only cleaned up at next cve fetch cycle.
```

Comment:
Actually, the cluster cve removal is handled through the fetcher for non-postgres scenario.